docs/docs/get_started/kubernetes_tutorial.md

```diff
@@ -716,152 +737,119 @@ Date: Thu, 23 Aug 2018 11:57:17 GMT
+              mountPath: /etc/secretless
+              readOnly: true
+      volumes:
+        - name: config
+          configMap:
+            name: quick-start-application-secretless-config
 ```
-<pre>
-Retrieving pets at Thu 23 Aug 2018 19:55:33 +08:
-HTTP/1.1 200
-Content-Type: application/json;charset=UTF-8
-Transfer-Encoding: chunked
-Date: Thu, 23 Aug 2018 11:55:33 GMT
 
-[{"id":1,"name":"Secretlessly Fluffy"}]
+### Networking
 
-...
-</pre>
+Since it resides in the same pod, the application can access the Secretless
+sidecar container over localhost.
 
-### Rotate Credentials
+As specified in the ConfigMap we created, Secretless listens on port
+`5432`, and hence this:
 
-You will be using **admin-credentials** to carry out these steps, pruning existing connections requires elevated privileges.
+```yaml
+          env:
+            - name: DB_URL
+              value: postgresql://localhost:5432/${APPLICATION_DB_NAME}?sslmode=disable
+```
 
-Begin by setting up environment variables (assumes the default setup with `minikube`):
+is all our application needs to locate Secretless.
 
-```bash
-export REMOTE_DB_ADMIN_USER=quick_start_admin_user
-export REMOTE_DB_ADMIN_PASSWORD=quick_start_admin_password
-export REMOTE_DB_URL=$(minikube ip):30001
+### SSL
 
-export APPLICATION_DB_USER=app_user
-# you can specify a different value for the new password below
-export APPLICATION_DB_NEW_PASSWORD=new_app_user_password
-```
+Notice the `?sslmode=disable` at the end of our `DB_URL`.
 
-#### Rotate Credentials In DB
-Now you can proceed to rotate the credentials in the database.
+This means that **the application connects to Secretless without SSL**, which
+is safe because it is intra-Pod communication over localhost.
 
-Remember, you will be using **admin-credentials** to carry out this step.
+However, the **connection between Secretless and Postgres is secure, and does
+use SSL**.  
+
+The situation looks like this:
 
-To rotate the application DB-user password, run this command:
-```bash
-docker run \
- --rm \
- -i \
- -e PGPASSWORD=${REMOTE_DB_ADMIN_PASSWORD} \
- postgres:9.6 \
-  psql \
-  -U ${REMOTE_DB_ADMIN_USER} \
-  "postgres://${REMOTE_DB_URL}/postgres" \
-  << EOSQL
-/* Rotate Application User password */
-ALTER ROLE ${APPLICATION_DB_USER} WITH PASSWORD '${APPLICATION_DB_NEW_PASSWORD}';
-EOSQL
 ```
-<pre>
-ALTER ROLE
-</pre>
+                 No SSL                       SSL
+Application   <---------->   Secretless   <---------->   Postgres
+```
 
-#### Update Credentials In Secret Store
+For more information on PostgreSQL SSL modes see:
 
-After rotation the password value held in the secret store is stale and requires updating.
-Run the following command to update the application-user DB-credentials password value in Kubernetes secrets:
+- [PostgreSQL SSL documentation](https://www.postgresql.org/docs/9.6/libpq-ssl.html)
+- [PostgreSQL Secretless Handler documentation](/docs/reference/handlers/postgres.html).
```

Comment:
this will be https://docs.secretless.io/Latest/en/Content/References/handlers/postgres.htm as well